File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1837.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
        
        int a = ler.nextInt(), b = ler.nextInt();
        int q = a/b, r = a%b;
        
        if(r<0){
            if(q<0) q--;
            else if(q>0) q++;
            else{
                if(a-((q-1)*b)>=0){
                    q--;
                } else{
                    q++;
                }
            }
            r = a-(q*b);
        }
        
        System.out.printf("%d %d\n", q, r);
        
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1837.kt
import java.util.*
import java.lang.Math

fun main(args: Array<String>) {
    val ler = Scanner(System.`in`)

    var a: Int = ler.nextInt();
    var b: Int = ler.nextInt();
    
    var q = a/b;
        var r = a%b;
        
        if(r<0){
            if(q<0) q-=1;
            else if(q>0) q+=1;
            else{
                if(a-((q-1)*b)>=0){
                    q-=1;
                } else{
                    q+=1;
                }
            }
            r = a-(q*b);
        }
        
        System.out.printf("%d %d\n", q, r);
}